src/rapids_singlecell/preprocessing/_harmony/__init__.py

```diff
@@ -153,6 +196,9 @@ def harmonize(
     correction_method
         Choose which method for the correction step: ``original`` for original method, ``fast`` for improved method. By default, use improved method.
 
+    use_gemm
+        If True, use a One-Hot-Encoding Matrix and GEMM to compute Harmony. If False use a label vector. This is more memory efficient and faster for large datasets with a large number of batches. Defaults to True for less than 30 batches.
```

Comment:
Suggested change for clarity.

```suggestion
        If True, use a One-Hot-Encoding Matrix and GEMM to compute Harmony. If False use a label vector. A label vector is more memory efficient and faster for large datasets with a large number of batches. Defaults to True for less than 30 batches.
```